addons/l10n_in/models/account_invoice.py

```diff
@@ -738,3 +739,16 @@ def _get_l10n_in_customer_invoice_title(self, proforma=False):
                 return _("Draft Proforma %(journal_name)s", journal_name=self.journal_id.name)
             elif self.state == 'cancel':
                 return _("Cancelled Proforma %(journal_name)s", journal_name=self.journal_id.name)
+
+    def _check_fiscal_country_vs_chart_template(self):
+        for record in self:
+            chart_template_map = self.env['account.chart.template']._get_chart_template_mapping()
+            chart_template = record.company_id.chart_template
+            fiscal_country = record.company_id.account_fiscal_country_id
+            if chart_template and chart_template == 'in':
+                curr_country_id = chart_template_map[chart_template]['country_id']
+                if fiscal_country.id != curr_country_id:
+                    raise ValidationError(_(
```

Comment:
```suggestion
                    raise ValidationError(self.env_(
```
Better to use `self.env._` because the it uses the current environment active language